openslides/agenda/static/js/agenda/base.js

```diff
@@ -0,0 +1,122 @@
+(function () {
```

Comment:
This other function should be coded as angular services.

I don't see the advantage to use two techincs to solve one problem.